surfsense_backend/alembic/versions/10_update_chattype_enum_to_qna_report_structure.py

```
@@ -0,0 +1,108 @@
+"""Update ChatType enum from GENERAL/DEEP/DEEPER/DEEPEST to QNA/REPORT_* structure
+
+Revision ID: 10
+Revises: 9
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
```

Ruff (0.11.9)

`sqlalchemy` imported but unused

Remove unused import: `sqlalchemy`

(F401)